src/assets/locales/fr.json

```diff
@@ -388,6 +440,15 @@
   "reaction-popover.emoji-label.smile": "Sourire",
   "reaction-popover.emoji-label.wave": "Faire coucou",
   "reaction-popover.title": "Réagir",
+  "revoke-token-modal.cancel": "Annuler",
+  "revoke-token-modal.description1": "Êtes-vous sûr de vouloir revoquer ce jeton ?",
+  "revoke-token-modal.description2": "Tout script ou requête utilisant ce jeton perdra l'accès.",
+  "revoke-token-modal.revoke": "Revoquer",
+  "revoke-token-modal.revoke-warning-1": "Cette action est",
+  "revoke-token-modal.revoke-warning-2": "permanente",
+  "revoke-token-modal.revoke-warning-3": "et",
+  "revoke-token-modal.revoke-warning-4": "ne peut être annulée.",
+  "revoke-token-modal.title": "Revoquer jeton",
```

Comment:
"revoke-token-modal.cancel": "Cancel",
  "revoke-token-modal.description1": "Are you sure you want to revoke this token?",
  "revoke-token-modal.description2": "Any scripts or requests relying on this token will lose access.",
  "revoke-token-modal.revoke": "Revoke",
  "revoke-token-modal.revoke-warning-1": "This action is",
  "revoke-token-modal.revoke-warning-2": "permanent",
  "revoke-token-modal.revoke-warning-3": "and",
  "revoke-token-modal.revoke-warning-4": "can not be undone.",
  "revoke-token-modal.title": "Revoke token",